tkg/client/management_components.go

```diff
@@ -120,7 +141,11 @@ func (c *TkgClient) getTKGPackageConfigValuesFile(managementPackageVersion, kube
 		return "", err
 	}
 
-	valuesFile, err := managementcomponents.GetTKGPackageConfigValuesFileFromUserConfig(managementPackageVersion, userProviderConfigValues, tkgBomConfig)
+	addonsManagerPackageVersion, err := c.getAddonsManagerPackageversion(managementPackageVersion)
```

Comment:
Can we also add an environment variable to override addonsManagerPackageVersion

Reply:
added _ADDONS_MANAGER_PACKAGE_VERSION